### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by imbstack**
The `--exit` mocha opt I added is explained in the commit message to d374a5b. If we would rather make this work better than just patch over an issue, I'm happy to go about making that happen as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,33 +15,33 @@
     "url": "https://github.com/taskcluster/taskcluster-auth.git"
   },
   "dependencies": {
-    "assume": "^1.3.1",
-    "aws-sdk": "^2.149.0",
```
Comment (djmitche) [inline package.json]: in the time between the last commit and this one, they ugpraded? :)

Comment (imbstack) [reply package.json]: Wow... looks like they did!

**Comment by djmitche** [comment]
Let's move fast and break things!!

**Comment by djmitche** [comment]
Reminder to self to merge https://github.com/taskcluster/taskcluster-docs/pull/225 once this lands.

**Comment by djmitche** [comment]
🍿

**Pull request merged**